packages/data/src/world/mm/pirate_fortress.yml

```diff
@@ -134,7 +135,7 @@
   exits:
     "Pirate Fortress Interior": "true"
     "Pirate Fortress Lone Guard": "true"
-    "Pirate Fortress Treasure Room Entry": "can_hookshot || short_hook_anywhere"
+    "Pirate Fortress Treasure Room Entry": "(can_hookshot || short_hook_anywhere) && can_evade_gerudo"
```

Comment:
seeing how hook lets you evade gerudo, the macro addition seems rather redundant here.

Reply:
Reverted